libs/api/shell/feature/src/lib/api-shell-feature.module.ts

```diff
@@ -3,11 +3,15 @@ import { ApiStorageApiFeatureModule as ApiStorageFeatureModule } from '@graduate
 import { Module } from '@nestjs/common';
 import {GraphQLModule} from '@nestjs/graphql'
 import { ApolloDriver, ApolloDriverConfig } from '@nestjs/apollo';
+import { ApiAccessStatusApiFeatureModule } from '@graduates/api/access-status/api//feature';
```

Comment:
```suggestion
import { ApiAccessStatusApiFeatureModule } from '@graduates/api/access-status/api/feature';
```
I think the double '//' is probably not intended

Reply:
Im looking into it as it was automatically done by vscode and it seems to create problems if it is a single slash

Reply:
Semantically they should be equivalent... 
Perhaps if its causing issues we should leave it be? @arneschreuder 
I just thought it looked weird.

Reply:
Nah, the double slashes cant be right.